presidio-analyzer/pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "presidio_analyzer"
+description = "Presidio Analyzer package"
+license = {text = "MIT License"}
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+requires-python = ">= 3.5"
```

Comment:
Why not 3.8?

Reply:
That's what we had in setup.py's, I don't know if this is actually prevents users to install with lower versions (that might work, though we are not supporting them officially)